build.sbt

```diff
@@ -32,13 +32,18 @@ commands += Command.command("ci-test") { s =>
     s
 }
 
-lazy val root = project
+// NOTE: We can also do this to run all matrix (crossScalaVersions * crossProject)
+addCommandAlias("ci-cross", "+ test; + publishLocal")
+
+lazy val root = crossProject(JVMPlatform, JSPlatform)
+  // .crossType(CrossType.Pure) // THIS IS NOT WORKING https://github.com/portable-scala/sbt-crossproject#configuration
+  // FIXME I add to create the 'shared' folder. Because we get erros like "not found: object zio"
```

Comment:
I'm not sure why this is not working on the project root ("." top level folder) already try a bunch of stuff

Is it ok to move `src` into the `shared` folder?

Reply:
FYI https://github.com/portable-scala/sbt-crossproject/issues/122

Please name this directory `core` (`core/src/main/....`).

[Build config example](https://github.com/typelevel/munit-cats-effect/blob/main/build.sbt).

Reply:
I always have cross-project in subdirectories.
Didn't know about that limitation. Thanks